server/controllers/stock/index.js

```diff
@@ -65,6 +65,10 @@ async function createStock(req, res, next) {
 
     const period = await Fiscal.lookupFiscalYearByDate(params.date);
     const periodId = period.id;
+
+    console.log('FISCAL_YEAR: _ID');
```

Comment:
It looks like you forgot some console.logs() here.

Curious that hound didn't catch them.

Reply:
It's Outdated